### ISSUE TITLE
(2 ^ 62) - 1 returns -1 on LLVM back-end

### ISSUE THREAD

**Issue by cgay** [dylan-lang/opendylan#1239]
`format-out("%d", (2 ^ 61) - 1)` gets an overflow error, as expected.

```
$ _build/bin/hello
Integer overflow: The result of the last operation is too large for this integer representation.
Backtrace:
  invoke-debugger:internal:dylan##1 + 0x29
  default-handler:dylan:dylan##1 + 0x12
  default-last-handler:common-dylan-internals:common-dylan##0 + 0x2e6
  signal:dylan:dylan##0 + 0x120
  error:dylan:dylan##0 + 0xb1
  -:dylan:dylan##1 + 0x16
  _Init_hello__X_hello_for_system + 0x6a
  _init + 0x151
  __libc_start_main + 0xf1
  _init + 0x62
```

`format-out("%d", (2 ^ 62) - 1)` prints `-1`.

```
$ _build/bin/hello
-1
```

**Comment by housel** [comment]
Fixed in PR#1256.

**Comment by pedro-w** [comment]
There is a compiler error when using the C back end, which I think was introduced by the regression test for this bug.
```
Cc /opendylan/_build/Bootstrap.3/build/libraries-test-suite-app/../libraries-test-suite/../common-dylan-test-suite/regressions.o : /opendylan/_build/Bootstrap.3/build/libraries-test-suite-app/../libraries-test-suite/../common-dylan-test-suite/regressions.c
:931:22: error: integer literal is too large to be represented in any integer type
  T0 = (dylan_value) 0xFFFFFFFFFFFFFFFF8000000000000001L;
                     ^
/opendylan/_build/Bootstrap.3/build/libraries-test-suite-app/../libraries-test-suite/../common-dylan-test-suite/regressions.c:932:31: error: integer literal is too large to be represented in any integer type
  MV_SET_ELT(0, (dylan_value) 0xFFFFFFFFFFFFFFFF8000000000000001L);
                              ^
2 errors generated.
```
